libs/masthead/vite.config.ts

```diff
@@ -0,0 +1,51 @@
+import { defineConfig } from "vitest/config";
+import react from "@vitejs/plugin-react";
+import checker from "vite-plugin-checker";
+import path from "node:path";
+import dts from "vite-plugin-dts";
+import peerDepsExternal from "rollup-plugin-peer-deps-external";
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  resolve: {
+    mainFields: ["module"],
+    dedupe: ["react", "react-dom"],
+  },
+  build: {
+    lib: {
+      entry: path.resolve(__dirname, "src/index.ts"),
+      name: "KeycloakMasthead",
+      formats: ["es", "umd"],
```

Comment:
Let's not build this as a UMD library.
```suggestion
      formats: ["es"],
```